TENDemo/AgoraManager/AgoraManager.swift

```diff
@@ -436,7 +438,7 @@ extension AgoraManager: AgoraRtcEngineDelegate {
     ///     - data: the data
     open func rtcEngine(_ engine: AgoraRtcEngineKit, receiveStreamMessageFromUid uid: UInt, streamId: Int, data: Data) {
         do {
-            let stt = try JSONDecoder().decode(STTStreamText.self, from: data)
+            let stt = try sttStreamDecoder.parseStream(data: data)
```

Comment:
This should also have a comment to explain that the Agora decoder is parsing the STT data